lib/strategy.js

```diff
@@ -14,6 +27,19 @@ function getHost(region) {
   }
 }
 
+/**
+ * `getMasheryHost` utility function
+ *
+ * Takes a two character region code and returns a Mashery host URL
+ *
+ *Examples:
+ *
+ *     getMasheryHost('us');  // returns 'us.api.battle.net'
+ *
+ * @param {String} region
+ * @return {String}
+ * @api protected
```

Comment:
Apologies. You're right, that was a mistake on my end. `protected` is accurate.

*sigh* still waking up it seems.